docs/about/contributing.md

```diff
@@ -7,21 +7,139 @@ nav_order: 2
+
+### General File Format
+
+**File format** for all files should be written in Markdown.
+
+- For help with Markdown syntax, see this [basic syntax guide](https://www.markdownguide.org/basic-syntax/)
+- To add links to other pages:
+    
+```
+# syntax
+[<text to click on>](<Github repo relative path to file>)
+
+# This is an example for a link to the Personas description page for 'Clinical Researcher':
+[Clinical Researcher](../personas/clinical-researcher.md)
+```
+    
+The **Site index** is automatically created by yaml headers in each file (only the title will be visible on the rendered website). See [template files](https://github.com/nih-cfde/usecases/tree/StyleGuide/docs/template_files) for example structure.
+
+  The `parent` yaml value depends on the website tab the Use Case file appears in:
+  
+  `parent:` value | Type
+  --- | ---
+  `Use Cases` | for the main Use Case file
+  `Personas` | for persona files
+  `Objectives` | for objective files
+  `User Tasks` | for user task files
+  `Requirements` | for requirements files
+  
+  The Use Case committee will finalize yaml header page numbers to ensure they do not clash with existing files.
+  
+**File names** should be lower case, have hyphens, and match the heading. Use Case (`ucXXXX`), User Task (`tXXXX`), and Requirement (`rXXXXX`) files should include a file ID numbered in order of appearance in this Use Case Repository website.
+
+   File type | Example
+   --- | ---
+   use case | uc0001-researcher-browse-and-filter.md
+   persona | clinical-researcher.md
+   objective | create-data-release.md
+   user task | t0001-access-cfde-interface.md
+   requirement | r00001-the-interface-will-support-gui-web-access-to-end-users.md
+   
+   The Use Case committee will finalize Use Case, User Task, and Requirement file IDs to ensure they do not clash with existing IDs.
+
+### `Use Case` files
+- Use Case titles always begin with their `UCXXXX` ID
+- Required sections:
+    - yaml index header
+    - Page header: title, date completed (`<month> <year>`), persona, objective
+    - `Summary`: a short summary of the use case
+    - `User Tasks`: a bullet point list of the tasks required for the use case, including links to each task page. Tasks are numbered by `TXXXX`.
+    - `Requirements`: a bullet point list of the requirements for each task, including links to each requirement page. Requirements are numbered by `RXXXXX`.
+- The persona, objective, user tasks, and requirements need to be linked to their corresponding description page (detailed in the following sections)
+- See [use case file template](https://github.com/nih-cfde/usecases/blob/StyleGuide/docs/template_files/use-case-template.md)
+
+### `Persona` files
+- Required sections:
+    - yaml index header
+    - One line description of the biological/computational experience and relation to any of the other Personas in the Use Cases Repository
```

Comment:
I honestly have no idea why I made two description sections in these...can you just merge the two descriptions together and take out the extra header line

Reply:
done! also went ahead and edited out the extra header lines in the existing persona files